robotpy_sphinx/all.py

```diff
@@ -7,5 +7,4 @@ def setup(app):
     from . import pybind11_fixer
 
     app.setup_extension(pybind11_fixer.__name__)
-    app.setup_extension("sphinx_autodoc_typehints")
```

Comment:
Just kidding, why did you get rid of this?

Reply:
See f39b20d76ae54c4620485b1073086a9e490392a0

Reply:
Does it do anything meaningfully bad though? We do have some (very few) non-pybind11 functions in various places where that could be useful.

Reply:
My opinion is that if we want a project to use it, then it should explicitly opt in. Apparently Sphinx 3.0 also added [a thing](https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#confval-autodoc_typehints) in this area too (which works differently to the plugin).